src/routes/land/[slug]/+page.svelte

```diff
@@ -0,0 +1,93 @@
+<script lang="ts">
+  import Accordion, { AccordionItem } from "$lib/components/Accordion";
+  import Button from "$lib/components/Button/Button.svelte";
+  import Card from "$lib/components/Card/Card.svelte";
+  import ContactCard from "$lib/components/ContactCard";
+  import ContentfulRichText from "$lib/components/ContentfulRichText";
+
+  import type { ServiceEntry, ServiceGroup } from "$lib/services/contentful/schema";
+
+  export let data;
+
+  // TODO: remove and validate core content pages with data once navigation changes are merged
+  // import pestControlData from "./pest-control-data.json";
+  // let data = pestControlData.data.serviceGroupCollection.items[0];
+
+  // import industrialHempData from "./industrial-hemp-data.json";
+  // let data = industrialHempData.data.serviceGroupCollection.items[0];
+  $: ({
+    title,
+    subheading,
+    description,
+    heroImage,
+    serviceListName,
+    serviceEntriesCollection,
+    contactInfoCollection,
+    additionalResources,
+  } = data);
+
+  let serviceEntries: ServiceEntry[] = [];
+  let serviceGroups: ServiceGroup[] = [];
+  $: serviceEntries = [];
+  $: serviceGroups = [];
+
+  $: {
+    data.serviceEntriesCollection?.items.map((item) => {
+      if (item && "entryTitle" in item) {
+        serviceEntries?.push(item);
+      } else if (item && "title" in item) {
+        serviceGroups?.push(item);
+      }
+    });
+  }
+</script>
+
+{#if heroImage}
+  <!-- TODO: Update this to use a proper Hero/Image Component 
+       (pending merge of https://github.com/la-ldaf/ldaf-site/pull/279) -->
+  <img src={heroImage.imageSource?.url} alt={heroImage.imageSource?.description} />
+{/if}
+<h1>{title}</h1>
+<p class="usa-intro">
+  {subheading}
+</p>
+{#if description}
+  <ContentfulRichText document={description?.json} />
+{/if}
+
+{#if serviceEntriesCollection}
+  <h2>{serviceListName}</h2>
+  {#if serviceEntries.length > 0}
+    {#each serviceEntries as item}
+      <Accordion multiselectable>
+        <AccordionItem title={item?.entryTitle}>
```

Comment:
Might be good to provide `id` here as well with the Service Entry's `sys.id` so that we don't need to rely on the default `Math.random` value in the component.

Reply:
Agreed. I'll keep the `Math.random` call in the component as a fallback in case `id` isn't supplied, but I added a clarifying comment and updated it to `sys.id` in this specific instance